src/IECoreGL/CurvesPrimitive.cpp

```diff
@@ -288,6 +288,18 @@ void CurvesPrimitive::renderMode( const State *state, bool &linear, bool &ribbon
 	ribbons = !state->get<UseGLLines>()->value();
 }
 
+bool CurvesPrimitive::renderUsesGLLines( const State *state ) const
+{
+	if( const auto s = state->get<UseGLLines>() )
+	{
+		if( s->value() )
+		{
+			return true;
+		}
+	}
+	return glslVersion() < 150;
```

Comment:
This all looks like it will work fine, but it does feel a little weird that it is so close to renderMode.  Why not just call renderMode and check the "linear" return ... I guess it could be perf critical so you want to skip the basis compare?  Why this check if `state->get<UseGLLines>()` is valid before dereferencing it, but renderMode doesn't?